modes/longitudinal/src/toolbarButtons.js

```diff
@@ -76,6 +74,68 @@ function _createSetToolActiveCommands(toolName) {
   return temp;
 }
 
+const StackImageSync = _createToggleButton(
+  'StackImageSync',
+  'link',
+  'Stack Image Sync',
+  [
+    {
+      commandName: 'toggleStackImageSync',
+    },
+  ],
+  'Enable position synchronization on stack viewports',
+  {
+    listeners: {
+      newStack: {
+        commandName: 'toggleStackImageSync',
+        commandOptions: { toggledState: true },
+      },
+    },
+  }
+);
+
+const StackPrefetch = _createToggleButton(
+  'StackPrefetch',
+  'link',
+  'Stack Prefetch',
+  [],
+  'Enable Complete Stack Prefetch',
+  {
+    listeners: {
+      newStack: {
+        commandName: 'stackPrefetch',
+      },
+    },
+    isActive: true,
+  }
+);
+
+const ReferenceLines = _createToggleButton(
+  'ReferenceLines',
+  'tool-referenceLines', // change this with the new icon
+  'Reference Lines',
+  [
+    {
+      commandName: 'toggleReferenceLines',
```

Comment:
We just recently removed the `toggleReferenceLines` command. Is there a way to avoid having to bring it back? Can the button instead just run the two commands and also have the listeners?

Reply:
What two commands would it run?

Reply:
Found them - will remove this method.